sample_apps/rso_sample_apps/nodejs/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "rso-example-app",
+  "version": "0.0.1",
+  "description": "rso example app",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Riot Tuxedo",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
I think it would be cool to have the "engines" here, specifying the nodejs version